### ISSUE TITLE
Volto does not generate correct Multiple Choice Edit Form from Python Schema

### ISSUE THREAD

**Issue by vincero** [plone/volto#952]
Example for Volto v. 4alpha:

```
class IMultipleChoiceExample(model.Schema):

    theoretical_informatics = schema.List(
        title=_("To which topics of Theoretical Informatics would you be able to answer questions?"),
        value_type=schema.Choice(values=(
            'Chomsky Hierarchy',
            'Automata',
            'Computability',
            'Complexity of Problems',
            'NP-Completeness')
            ),
        required=False,
    )
```

Generated Form from Classical Plone (Correct!):

![mc-classical](https://user-images.githubusercontent.com/1233101/67619223-2a4b3500-f7f9-11e9-8a2b-555f128cf1ca.png)

Generated Form from Volto (no selectable fields):

![mc-volto](https://user-images.githubusercontent.com/1233101/67619221-29b29e80-f7f9-11e9-9ff6-adfdf31ed244.png)

Generated Form from Classical Plone displayed in Volto:

![mc-classical-in-volto](https://user-images.githubusercontent.com/1233101/67619222-2a4b3500-f7f9-11e9-9b82-b4d9add5759f.png)

**Comment by sneridagh** [comment]
@vincero should work in latest alpha.

**Comment by pbauer** [comment]
This is still broken in Volto 5.0.1

![options](https://user-images.githubusercontent.com/453208/79871244-5d573580-83e4-11ea-8153-d98aeb2c2647.png)

![selected](https://user-images.githubusercontent.com/453208/79871434-9f807700-83e4-11ea-9bb7-6af1baf6d1f0.png)

![inspect](https://user-images.githubusercontent.com/453208/79871250-5f20f900-83e4-11ea-8782-1f0908534ffa.png)

**Comment by pbauer** [comment]
With the newest `plone.restapi = 6.6.0` the error is stil there but a but different:

![options](https://user-images.githubusercontent.com/453208/79872339-f76bad80-83e5-11ea-94ab-aec7ec81d998.png)

![inspect](https://user-images.githubusercontent.com/453208/79872412-0ce0d780-83e6-11ea-979c-0492b50e536d.png)

**Comment by sneridagh** [comment]
This should have fixed it (I tested it out):
https://github.com/plone/volto/blob/c13ee5693a4dade5c1c38578ee4339e04902b474/src/components/manage/Widgets/SelectWidget.jsx#L337-L342

For Select fields, might be that the type of fields that it's being rendered is different? So the resultant widget does not have the fix?

**Comment by sneridagh** [comment]
I remember even test it with your training package during the Alpine sprint.

**Comment by sneridagh** [comment]
Just tested out:
https://volto.kitconcept.com/philip-choice-test

<img width="410" alt="image" src="https://user-images.githubusercontent.com/486927/79890753-3eff3300-8400-11ea-81f8-0083fb312929.png">

It's a TTW added field, might be that changes from the programmatic one, but now I can't tell how.
Are you sure you are on Volto 5.0.1?

**Comment by pbauer** [comment]
Yes, I am on Volto 5.0.1 with restapi 6.6.0. You use the sidebar (blocks layout). I use the default layout without blocks.

**Comment by sneridagh** [comment]
Fixed by: https://github.com/plone/volto/pull/1405

**Comment by sneridagh** [comment]
@pbauer I will cut a release this evening. Tell me if it's fine now!

**Comment by sneridagh** [comment]
@pbauer released Volto 5.1.0
